packages/plugin-compat/sources/extensions.ts

```diff
@@ -52,4 +52,12 @@ export const packageExtensions: Array<[string, any]> = [
       [`@types/keyv`]: `^3.1.1`,
     },
   }],
+  // vscode-jsonrpc and vscode-languageserver-protocol are imported at runtime
+  // yet absent from typescript-language-server's dependencies
```

Comment:
Can I ask you to open a PR as well on `typescript-language-server` and reference it in the comment? I'd like to keep track of which builtin compat entries are fixed upstream 🙏

Reply:
I actually considered that, but never got around to do it. But, absolutely, I'll do it right away :)